pdfpc/pdfpc.sty

```diff
@@ -90,6 +90,7 @@
   \def\PDFPC@notesposition{auto}%
 \fi%
 %
+\def\pdfpcsetup{\kvsetkeys{PDFPC}}%
```

Comment:
Is this change worth increasing the version number of the package?

Reply:
Yes, but let's first try to resolve the original problem.

Reply:
OK, let's release it as is. I have too many things to finish currently......